src/kibana4-dashboards/dashboard/CF-App-ERRORs.json.erb

```diff
@@ -0,0 +1,62 @@
+<% require 'json' %>
+{
+  "title": "CF App ERRORs",
+  "hits": 0,
+  "description": "",
+  "panelsJSON": "<%= JSON.parse(<<ENDOFFJSON).to_json.gsub(/"/){ '\"' }
```

Comment:
A bit of magic ERB to turn the pretty printed json back into the json encoded and escaped string the ES wants